.github/workflows/ci.yml

```diff
@@ -148,3 +202,30 @@ jobs:
               LEGO1/omni/src/video/flic.cpp \
               $action_headers \
             --path LEGO1/omni LEGO1/lego/legoomni
+
+  release:
+    name: 'Release'
+    if: ${{ github.event_name == 'push' }}
```

Comment:
doesn't it need a check whether we are on the master branch as well? otherwise it would trigger on all branch pushes?